docs/docs/userguide/INSTALLATION.md

```diff
@@ -125,11 +130,13 @@ By default, the Helm charts will not configure the products for Data Center clus
 !!!warning "Bamboo clustering"
     Because of the limitations outlined under [Bamboo and clustering](../troubleshooting/LIMITATIONS.md#cluster-size) the `clustering` stanza is not available as a configurable property in the Bamboo `values.yaml`.
 
+!!!warning "Crowd clustering"
```

Comment:
Is there a specific reason why this is on by default for Crowd? Other products have clustering disabled by default.

Reply:
During the installation, there's no way to choose if clustering is enabled, and I just find it enabled in crowd.cfg.xml after the installation is completed:

```
    <property name="crowd.clustering.enabled">true</property>
```
[Docs](https://confluence.atlassian.com/crowd/installing-crowd-data-center-935369773.html#InstallingCrowdDataCenter-3.AddthefirstCrowdnodetoyourloadbalancer) say, clustering must be enabled first, but this isn't the behavior I see.